src/main.js

```diff
@@ -26,7 +26,7 @@ function NoMusk(KeyWords) {
 
 /* Read files from extention filesystem */
 async function readExtentionFile(fileName) {
-	return (await (await fetch(chrome.runtime.getURL(fileName))).text()).split("\n").filter((word) => word.length > 0);
```

Comment:
file name being passed in wasn't changed but the extension did. I'll approve but I think I'll rebase against your branch instead of merging this